### DISCUSSION TITLE
`describeTag` is @lerna-lite/version is not working as expected

### DISCUSSION THREAD

**Discussion by izikl** [ghiscoding/lerna-lite#820] [Q&A] [answered]
I added the following to lerna.json:
```
"version": {
  "describeTag": "packages-v*"
}
```

When I was running `lerna version --no-push --yes --exact` I had the following tags in the repo (in that order):
1. some-tag-1
2. some-tag-2
3. some-tag-3
4. packages-v2.2.2

I was expecting that the conventional commits will run on commits from tag `packages-v2.2.2`, since `describeTag` was set to `packages-v*`, however, it ran on commits since the first tag - `some-tag-1`. 

Is this a bug, or am I running it incorrectly?

**Comment by ghiscoding** [comment]
I don't think that the code changed from the original implementation, it came from the original repo. I would suggest that you search in the original Lerna issues and/or contribute a fix. 

ahh wait, there was this PR #515 made by another user that had changes to `describeTag`, could that be related? I have never used this feature myself and you probably know more about the feature than I do... There are some examples in the unit tests in that same PR, it might be helpful to you

**Reply by izikl** [reply, marked as answer]
I figure it out. Moved `describeTag` in lerna.json from root->command->version to root

From
```json
{
    "command": {
        "version": {
            "describeTag": "packages-v*"
        }
    },
}

```
 to
```json
{
    "describeTag": "packages-v*",
}
```

**Reply by ghiscoding** [reply]
hmm that's a bit odd though since the PR #515 is really adding this as a `lerna version` option, the JSON Schema was also updated with that in mind. However looking at some of the code, you seem to be right since `const describeTag = this.project.config.describeTag;` seems to be pointing to the root config, the same goes for this [test](https://github.com/lerna-lite/lerna-lite/pull/515/files#diff-00c12c403c7c2f73fdc127603332510b7badba4197d89a77c67228dbcfd4fbadR743-R746). We should probably makre sure the JSON Schema points at the root for this option and/or support `describeTag` from both the root and the `version` option. 

Were you using the Lerna-Lite [JSON Schema](https://github.com/lerna-lite/lerna-lite?tab=readme-ov-file#json-schema) or not? 

EDIT

actually the JSON Schema was incorrect since it was showing `describeTag` to be only available in the version command and not in the root. Though I did fix it in the PR mentioned below, it is now accepted in both the root and the version command

**Reply by ghiscoding** [reply]
ok I found the issue and opened a new issue #826 from your discussion, I just had to modify this line (PR #827 opened) to fix the issue you reported
```diff
- this.project.config.describeTag;
+ this.options.describeTag; 
```
calling `this.options` works with both the config root and/or the command  (`command.version.describeTag`)
